src/ui/public/agg_types/buckets/date_histogram.js

```diff
@@ -41,7 +41,8 @@ export default function DateHistogramAggType(timefilter, config, Private) {
     makeLabel: function (agg) {
       const output = this.params.write(agg);
       const params = output.params;
-      return params.field + ' per ' + (output.metricScaleText || output.bucketInterval.description);
+      const field = params.field || agg.params.field.displayName || '';
```

Comment:
@Bargs I'm comfortable with this approach if we want to roll it out for 5.x, since that'll be given a full blown QA run, but if we want to get this into 5.0, we should be more cautious around the access here.